src/curvefit/models/core_model.py

```diff
@@ -159,3 +186,12 @@ def bounds(self):
         re_bounds = np.repeat(re_bounds[None, :, :], self.data_inputs.num_groups, axis=0)
 
         return np.vstack([fe_bounds, re_bounds.reshape(self.param_set.num_fe * self.data_inputs.num_groups , 2)])
+
+    @property
```

Comment:
@zhengp0 please do check this part and the bounds part above! I was not sure if the order is correct.

Reply:
Seems right to me!